src/dparse/parser.d

```diff
@@ -188,6 +188,24 @@ class Parser
         return attachCommentFromSemicolon(node, startIndex);
     }
 
+    /**
+     * Parses an AliasAssign.
+     *
+     * $(GRAMMAR $(RULEDEF aliasAssign):
+     *       $(RULE aliasInitializer)
```

Comment:
this documentation says it would be syntactically valid to use templateParameters in an assignment. As it's never the case that it would actually be valid code I think it would be better to define a proper grammar here.